pkg/cmd/server/bootstrappolicy/infra_sa_policy.go

```diff
@@ -65,8 +65,8 @@ const (
 	ServiceLoadBalancerControllerRoleName                = "system:service-load-balancer-controller"
 
 	// TODO can we just rename these or how do we handle upgrades?
-	InfraPetSetControllerServiceAccountName = "pet-set-controller"
-	PetSetControllerRoleName                = "system:pet-set-controller"
+	InfraStatefulSetControllerServiceAccountName = "pet-set-controller"
```

Comment:
> @deads2k @liggitt can we rename the SA here? What's the recommended approach for those? Currently (as of 1.5) there's no such resource as PetSet, no migration was provided, either. Any ideas, suggestions how to proceed?

Remove the old from here and then specify the new.  The old permissions will rot, but not be removed on customer systems until they run a manual role reconcile, but that's ok.

Also, match names with upstream if possible.